lib/express-rate-limit.js

```diff
@@ -10,6 +10,7 @@ function RateLimit(options) {
       statusCode: 429, // 429 status = Too Many Requests (RFC 6585)
       headers: true, //Send custom rate limit header with limit and remaining
       draft_polli_ratelimit_headers: false, //Support for the new RateLimit standardization headers
+      precision: 4, //Number of request buckets inside time window. Buckets serve as a performant alternative to tracking timestamp of each request and clearing them one by one.
```

Comment:
Lets make the default `1`, otherwise this could end up surprising some people. I might bump it up to 4 in the next major release, though.